packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1207,6 +1206,21 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     }
   };
 
+  private _onRenderLabel = (props: IComboBoxProps): JSX.Element | null => {
```

Comment:
Since the rendering depends on `this._multiselectAccessibleText`, it feels like it should be a parameter to this function and not an instance variable. That way when a user overrides this function they can access all necessary state.